frontend/admin/src/js/components/Table/Table.tsx

```diff
@@ -287,16 +287,16 @@ function Table<T extends Record<string, unknown>>({
                   <div data-h2-flex-item="base(content)">
                     <Button type="button" mode="inline" color="white">
                       {intl.formatMessage({
-                        defaultMessage: "Download XML",
+                        defaultMessage: "Download CSV",
```

Comment:
You'll also need to remove the existing French string
```
  "tJTEzl": {
    "defaultMessage": "Télécharger XML",
    "description": "Text label for button to download an xml file of items in a table."
  },
```
New French string defaultMessage will be **Télécharger CSV**

Reply:
Thanks for getting that translation for me, update with [update French csv button label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3695/commits/246306b2f0c253fb7b314acb7144518f194671c7)